indico/modules/events/management/__init__.py

```diff
@@ -21,6 +21,8 @@
 from indico.core import signals
 from indico.core.config import Config
 from indico.modules.events.management.util import can_lock
+from indico.modules.events.models.events import Event
```

Comment:
You are not using this import and it breaks the tests (some weird circular import that only happens while running the tests): https://travis-ci.org/indico/indico/builds/114722522